spec/traverse_sequences_spec.rb

```diff
@@ -127,13 +127,11 @@
     end
 
     it 'has done markers differently-sized sequences' do
-      expect(@callback_s1_s2.done_a).to eq([['p', 9, 's', 10]])
+      expect(@callback_s1_s2.done_a).to eq([['p', 9, 't', 11]])
```

Comment:
Here are the 2 sequences we're comparing:
![expose](https://user-images.githubusercontent.com/12286472/146670709-203b2b71-d3f2-47a7-8801-a0f09964a504.png)

the last element that they'll match is `m`, after that, `seq1` has 2 elements left, `seq2` has 3 elements left.
So when `finished_a` is called, it should be called with the last element of `seq1`, and the last element of `seq2`, which is `p` and `t`, with their corresponding indexes.

Reply:
Yes.